### PR TITLE
Mark sktm.patchwork/db private methods

### LINKED ISSUE #12: Mark all private class methods as such
To avoid exposing too much interface, prepend all private class method names with `__` to have them recognized as such.

### PR DESCRIPTION
**Pull request by danghai**
I make PR to mark sktm.patchwork private methods
Fix #12

### CODE REVIEW

**Comment by danghai** [comment]
@spbnick right `get_patch_by_id()` should be public. However, `set_patch_check()` should be private because there are same method name `set_patch_check()`, but difference argument. 
I fixed it. @spbnick @veruu  Could you review it? Thanks

**Comment by danghai** [comment]
I also mark the change for `db.py` too

File: sktm/patchwork.py [hunk 1]
```diff
@@ -291,13 +291,13 @@ def __init__(self, baseurl, project_name, skip, is_rh_fork=False):
                           default.
         """
         self.baseurl = baseurl
-        self.project_id = self.get_project_id(project_name)
+        self.project_id = self.__get_project_id(project_name)
```
Comment (veruu) [inline sktm/patchwork.py]: This method is defined in child classes so I believe this change would break things.

File: sktm/patchwork.py [hunk 2]
```diff
@@ -330,7 +330,7 @@ def get_patch_message(self, patch_id):
 
         return email.message_from_string(response.content)
 
-    def get_header_values_all(self, patch_id, *name_tuple):
+    def __get_header_values_all(self, patch_id, *name_tuple):
```
Comment (veruu) [inline sktm/patchwork.py]: I wouldn't mark this method as private. Right now, it's not used outside of parent class methods, but both this and `get_header_values_first` are meant to be freely used in child classes.

**Comment by danghai** [comment]
@veruu I resolve the conflict.

**Comment by danghai** [comment]
@veruu Sorry, I do not see this method. I fix it!

File: sktm/db.py [hunk 3]
```diff
@@ -524,31 +525,31 @@ def commit_patch(self, patch_id, patch_name, patch_url, baseurl,
...
+        source_id = self.__get_sourceid(baseurl, project_id)
         self.cur.execute('INSERT OR REPLACE INTO patch(id, name, url, '
                          'patchsource_id, date) '
                          'VALUES(?,?,?,?,?)',
                          (patch_id, patch_name, patch_url, source_id,
                           patch_date))
         self.conn.commit()
 
-    def commit_series(self, patches):
+    def __commit_series(self, patches):
```
Comment (veruu) [inline sktm/db.py]: The recent changes call this method from outside, so this marking isn't valid anymore and will break things.

Comment (danghai) [reply sktm/db.py]: Sorry, I do not notice it. I will fix it. Thanks

Comment (danghai) [reply sktm/db.py]: I should look your previous patch after resolving the conflict

**Comment by danghai** [comment]
@veruu I fix it! Coud you review it now?

**Comment by danghai** [comment]
@veruu haha hopefully :)

**Pull request merged**